README.md

```diff
@@ -62,7 +62,7 @@ challenges in mind with the proposal that follows.
 
 ## A Proposal
 
-By default, servers should receive only the user agent's brand and major version number. Servers
+By default, servers should receive only the user agent's brand and significant version number. Servers
```

Comment:
I like the "significant" wording, fitting into the "rolling update" system most modern browsers take.

Reply:
👍